test/run.js

```diff
@@ -43,7 +43,7 @@ module.exports = file => {
   describe(cmd, () => {
     fancy
     .retries(CI ? 1 : 0)
-    .timeout(600000)
+    .timeout(600_000)
```

Comment:
Does this work? 🤔

Reply:
Yes it does, its part of the language.

Reply:
Wow, TIL: https://github.com/tc39/proposal-numeric-separator
An apparently it's been available in node since [version 12.5.0](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#notable-changes-40).